src/sandstorm/app-index/app-index.c++

```diff
@@ -288,6 +321,7 @@ public:
       } else if (path.startsWith("reject/")) {
         indexer.reject(path.slice(strlen("reject/")),
             kj::str(params.getContent().getContent().asChars()));
```

Comment:
No CSRF protection except the Sandstorm built-in one. You're bold! I like that! This is not an objection.